src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -44,6 +45,7 @@ type Props = {
     getAvatarUrl: GetAvatarUrlCallback,
     getMentionWithQuery?: Function,
     getUserProfileUrl?: GetProfileUrlCallback,
+    hasNewThreadedReplies?: boolean,
```

Comment:
Should we add a unit test to ActiveState.test.js to test for this condition. Basically to verify that the right component is rendered based on the value of that boolean prop.  We wanna make sure that nothing breaks and accidentally causes the new BaseComment to show.

Reply:
Good point, added test